internal/server/http/handler/v1/treecluster/handler.go

```diff
@@ -28,12 +29,12 @@ var (
 // @Router			/v1/cluster [get]
 // @Param			page	query	string	false	"Page"
 // @Param			limit	query	string	false	"Limit"
-// @Param			status	query	string	false	"Status"
 // @Security		Keycloak
 func GetAllTreeClusters(svc service.TreeClusterService) fiber.Handler {
 	return func(c *fiber.Ctx) error {
 		ctx := c.Context()
-		domainData, err := svc.GetAll(ctx)
+
+		domainData, totalCount, err := svc.GetAll(ctx)
```

Comment:
Is `totalCount` not the same as `len(domainData)`? Is there a reason why you return a seperate value? So I would suggest this


```suggestion
		domainData, err := svc.GetAll(ctx)
		totalCount := len(domainData)
```

Reply:
The domainData is the paginated treeCluster data isn't it? That's not the `totalCount` of all tree cluster

Reply:
True, sorry my Bad 🫣